src/refactor/inputHandler.py

```diff
@@ -0,0 +1,26 @@
+'''
+ * Copyright (C) 2022  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+'''
+'''
+ * @file inputHandler.py
+ * @author Robert Taylor
+ * @date 2022-09-21
```

Comment:
Are you actually going to write this in September? ;->

Reply:
Haha whoops, I incremented the header I copied the wrong way